testdata/default/inline/FuzzInlineConf.t.sol

```diff
@@ -12,3 +12,15 @@ contract FuzzInlineConf is DSTest {
         require(true, "this is not going to revert");
     }
 }
+
+/// forge-config: default.fuzz.runs = 500
```

Comment:
could we decrease this for ci, we just want to assert that runs match, so could decrease significantly